From 846f1e2b8a1499341cd0af29e3a3ad1dcd31e9de Mon Sep 17 00:00:00 2001
From: Michael Park <mpark@apache.org>
Date: Thu, 15 Oct 2015 19:54:02 -0700
Subject: [PATCH] Set LIBPROCESS_IP into docker container.

Signed-off-by: Gilbert Song <songzihao1990@gmail.com>
---
 src/docker/docker.cpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/docker/docker.cpp b/src/docker/docker.cpp
index d13869db7..d24f506e6 100644
--- a/src/docker/docker.cpp
+++ b/src/docker/docker.cpp
@@ -681,6 +681,11 @@ Try<Docker::RunOptions> Docker::RunOptions::create(
   options.env["MESOS_SANDBOX"] = mappedDirectory;
   options.env["MESOS_CONTAINER_NAME"] = name;
 
+  Option<string> libprocessIP = os::getenv("LIBPROCESS_IP");
+  if (libprocessIP.isSome()) {
+    options.env["LIBPROCESS_IP"] = libprocessIP.get();
+  }
+
   if (resources.isSome()) {
     // Set the `MESOS_ALLOCATION_ROLE` environment variable. Please note
     // that tasks and executors are not allowed to mix resources allocated
-- 
2.25.1

