From 0e60d8262a9c945121a92cd2e0137335cf9fa6b1 Mon Sep 17 00:00:00 2001
From: Till Toenshoff <toenshoff@me.com>
Date: Sat, 23 Nov 2019 02:31:22 +0100
Subject: [PATCH] Set LIBPROCESS_IP into docker container.

---
 src/docker/docker.cpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/docker/docker.cpp b/src/docker/docker.cpp
index d13869db7..d24f506e6 100644
--- a/src/docker/docker.cpp
+++ b/src/docker/docker.cpp
@@ -681,6 +681,11 @@ Try<Docker::RunOptions> Docker::RunOptions::create(
   options.env["MESOS_SANDBOX"] = mappedDirectory;
   options.env["MESOS_CONTAINER_NAME"] = name;
 
+  Option<string> libprocessIP = os::getenv("LIBPROCESS_IP");
+  if (libprocessIP.isSome()) {
+    options.env["LIBPROCESS_IP"] = libprocessIP.get();
+  }
+
   if (resources.isSome()) {
     // Set the `MESOS_ALLOCATION_ROLE` environment variable. Please note
     // that tasks and executors are not allowed to mix resources allocated
-- 
2.23.0

