diff -Nwrup openresty-1.11.2.4.orig/configure openresty-1.11.2.4/configure
--- openresty-1.11.2.4.orig/configure	2017-08-18 17:56:10.525812446 +0200
+++ openresty-1.11.2.4/configure	2017-08-18 17:57:36.720556459 +0200
@@ -626,28 +626,6 @@ _END_
             $luajit_xcflags .= " -DLUAJIT_ENABLE_LUA52COMPAT";
         }
 
-        {
-            # check -msse4.2
-            my ($out, $cfile) = tempfile("resty-config-XXXXXX",
-                                         SUFFIX => '.c', TMPDIR => 1,
-                                         UNLINK => 1);
-
-            print $out "int main(void) { return 0; }";
-            close $out;
-
-            my $ofile = tmpnam();
-            my $comp = ($cc || 'cc');
-
-            if (system("$comp -o $ofile -msse4.2 -c $cfile") == 0 && -s $ofile) {
-                print "INFO: found -msse4.2 in $comp.\n";
-                $luajit_xcflags .= " -msse4.2";
-                unlink $ofile;
-
-            } else {
-                print "WARNING: -msse4.2 not supported in $comp.\n";
-            }
-        }
-
         if ($opts->{debug}) {
             $luajit_xcflags .= " -DLUA_USE_APICHECK -DLUA_USE_ASSERT";
             $luajit_xcflags =~ s/^ +//;
